### DISCUSSION TITLE
more Types

### DISCUSSION THREAD

**Discussion by MikeCZ23** [element-plus/element-plus#16934] [Q&A]
how can I add more variants?
(primary, danger, warning, info, CUSTOM, CUSTOM1,...)

adding a new color palette I can handle it no problem, I just don't want to overwrite the existing ones

**Comment by warmthsea** [comment]
Not currently supported.

**Reply by MikeCZ23** [reply]
Ok thx

**Comment by MikeCZ23** [comment]
hmm. If I put it in ``div``, it can be bypassed. except the description text can't change the color

**Comment by MikeCZ23** [comment]
![Snímek obrazovky 2024-05-24 121841](https://github.com/element-plus/element-plus/assets/45541448/f46456a1-bc38-4297-aa90-f43b24eb2f3e)
on text no work, bad work
